orcestra_book/ec_track.md

```diff
@@ -0,0 +1,71 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# EarthCARE Validation during ORCESTRA
+
+EarthCARE regularly flew over the measurement area, allowing each of the aircraft to fly underneath. The data from the underflights are used to validate the EarthCARE measurements. This page shows how to compare EarthCARE and HALO tracks.
+
+First, we load the track data. EarthCARE tracks are available for different regions of interest (`roi`) and forecast days.
```

Comment:
Could we add here, that these are the tracks we've used during planning? - I guess that since earthcare data should now be "public", there should be other ways to obtain (at least past) tracks, maybe even measured tracks.